boilerplate/src/configs/knexfile.js

```diff
@@ -0,0 +1,44 @@
+// Update with your config settings.
+
+/**
+ * This file will be used when running knex migrations
+ */
+
+module.exports = {
+
+  // for development environment
+  development: {
+    client: 'postgresql',
+    connection: {
+      database: 'dev_db',
+      user: 'postgres',
+      password: 'postgres',
```

Comment:
There has to be a better way than storing the password in plaintext in a JS file. First thing that comes to mind are environment variables that the container loads on startup from somewhere. But this might be out of the scope of this pull request, it's a similar issue to how I store credentials in the db code. @kmyn what do you think?